purview/models/koji.py

```diff
@@ -34,3 +35,10 @@ class KojiTask(StructuredNode):
     priority = IntegerProperty()
     arch = StringProperty(required=True)
     method = StringProperty(required=True)
+
+
+class KojiTag(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    creator = RelationshipTo('.user.User', 'CREATED_BY')
```

Comment:
Can you use `owners` here for consistency with our other models? Then the relationship name in Neo4j should be `OWNED_BY`.

Reply:
On second thought, maybe we don't need this data at all.

Reply:
Sure, I can remove that relationship

Reply:
Okay, if you do can you remove the inverse as well?

Reply:
Absolutely!, now please enjoy your sunday night. you can review this tomorrow :)